### PR TITLE
[Django 3.1] Upgraded Django to the latest version(3.1.7) till now

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- Django==3.0.12 to Django==3.1.7 (latest version as of now)

### CODE REVIEW

**Review by romainr — APPROVED** [review]
:+1: Congratz!

File: desktop/core/src/desktop/lib/django_util.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 
 from django.conf import settings
 from django.core import serializers
+from django.core.exceptions import FieldDoesNotExist
```
Comment (agl29) [inline desktop/core/src/desktop/lib/django_util.py:29]: @romainr  please check it also.

Comment (romainr) [reply desktop/core/src/desktop/lib/django_util.py:29]: Will it brake older Django?

Comment (agl29) [reply desktop/core/src/desktop/lib/django_util.py:29]: 1- > You can see the from `django.core.exceptions import FieldDoesNotExist  # NOQA` @ https://docs.djangoproject.com/en/1.11/_modules/django/db/models/fields/

2-> https://docs.djangoproject.com/en/3.1/releases/3.1/#id1 at this link we see that `The compatibility import of django.core.exceptions.FieldDoesNotExist in django.db.models.fields is removed.`

3-> And in the last one we can see that source of `FieldDoesNotExist` for both Django ( 1 and 3) is same 
        1->https://docs.djangoproject.com/en/3.1/_modules/django/core/exceptions/#FieldDoesNotExist (for Django 3.1)
        2->https://docs.djangoproject.com/en/1.11/_modules/django/core/exceptions/#FieldDoesNotExist (for Django 1.11)

Comment (romainr) [reply desktop/core/src/desktop/lib/django_util.py:29]: :+1: Perfect, very clear!

**Review by romainr — APPROVED** [review]

**Pull request merged**